fhir-persistence/src/main/java/com/ibm/fhir/persistence/context/FHIRPersistenceContextFactory.java

```diff
@@ -30,7 +30,8 @@ private FHIRPersistenceContextFactory() {
      * @param event the FHIRPersistenceEvent instance to be contained in the FHIRPersistenceContext instance
      */
     public static FHIRPersistenceContext createPersistenceContext(FHIRPersistenceEvent event) {
-        return new FHIRPersistenceContextImpl(event);
+        return FHIRPersistenceContextImpl.builder(event)
```

Comment:
I wonder if we can deprecate the factory and move callers over to use the builder directly...

Reply:
Yeah, didn't want to make too many unrelated changes, but I think we should move to using the builder when it's convenient.

Reply:
opened https://github.com/IBM/FHIR/issues/3157 for this, but agree its quite minor